tests/c/cscore_tests.cpp

```diff
@@ -39,7 +39,7 @@ void cscore(CSOUND *cs)
 // TEST (CScoreTests, testCscore)
```

Comment:
seems like this file can be deleted if cscore is removed

Reply:
done